mmv1/products/documentaiwarehouse/Location.yaml

```diff
@@ -0,0 +1,96 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Location'
+base_url: 'projects/{{project}}/locations/{{location}}:initialize'
+# self_link: '{{name}}'
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/document-warehouse/docs/overview'
+  api: 'https://cloud.google.com/document-warehouse/docs/reference/rest/v1/projects.locations'
+description: |
+  A location is used to initialize a project.
+autogen_async: true
+async: !ruby/object:Api::OpAsync
+  actions: ['create'] 
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms:
+      1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+immutable: true
+skip_read: true
+skip_delete: true
+skip_sweeper: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "document_ai_warehouse_location"
+    primary_resource_id: "example"
```

Comment:
With the property `exclude_import: true` on the resource, `skip_import_test: true` is needed for the example.

Reply:
`exclude_import: true` has not been set...

Reply:
Yeah, it looks like the resource needs more properties. https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/api/resource.rb
```
id_format: 'projects/{{project}}/locations/{{location}}'
exclude_import: true
```